docs/guides/typescript.md

```diff
@@ -223,8 +397,12 @@ type Event =
       type: 'EVENT_WITHOUT_FLAG';
     };
 
-createMachine<Context, Event>(
+createMachine(
   {
+    schema: {
+      context: t<Context>(),
```

Comment:
just an extra argument for `{} as ...` - if somebody lands on a doc site which doesnt mention `import { t } from 'xstate'` they might not be able to easily find our what that `t` is and how to get it. Using cmd/ctrl+F: "t" won't help them much either 😉

Reply:
@davidkpiano, what do you think of `t()`?